### PR TITLE
Added the 103 validation rule.

### LINKED ISSUE #14: Error code 103 - The ethnicity code is either not valid or has not been entered.
#### Error code
103
#### Description to display
The ethnicity code is either not valid or has not been entered.
#### DfE Long-form description of error
The ethnic origin field must not be blank, and code must be taken from the list supplied in the DfE children looked after collection guide.  Valid ethnic origin codes are WBRI, WIRI, WOTH, WIRT, WROM, MWBC, MWBA, MWAS, MOTH, AIND, APKN, ABAN, AOTH, BCRB, BAFR, BOTH, CHNE, OOTH, REFU, NOBT.  Check that a valid ethnic origin code has been entered, taken from the list supplied in the DfE children looked after collection guide.  The old ethnicity codes used in years prior to 2009 are no longer valid codes.
#### DfE suggested coding
`<ETHNIC>` must be provided and be a valid value  For allowed values please see associated code set.

For further info, refer to the DfE validation document [here](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/974999/Children_looked_after_data_collection_2020_to_2021_validation_checks_-_version_1.3.pdf)

### PR DESCRIPTION
**Pull request by jfan2006**
Added the ethnicity checking for 103 validation rule.

Closes #14

### CODE REVIEW

File: validator903/validators.py [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+        description='The ethnicity code is either not valid or has not been entered.',
+        affected_fields=['ETHNIC'],
+    )
+
+    def _validate(dfs):
+        if 'Header' not in dfs:
+            return {}
+        
+        header = dfs['Header']
+        code_list = ['WBRI', 'WIRI', 'WOTH', 'WIRT', 'WROM', 'MWBC', 'MWBA', 'MWAS', 'MOTH', 'AIND', 'APKN', 'ABAN', 'AOTH', 'BCRB', 'BAFR', 'BOTH', 'CHNE', 'OOTH', 'REFU', 'NOBT']
```
Comment (mwaddoups) [inline validator903/validators.py]: Just a note here - when writing longer lists, it's usually better to break them over multiple lines as it makes the code easier to read.

So for this, I would consider
```
code_list = [
    'WBRI',
    'WIRI',
    'WOTH',
    # etc...
]
```

Comment (mwaddoups) [reply validator903/validators.py]: Up to you if you want to fix this before I merge the PR - let me know.

Comment (jfan2006) [reply validator903/validators.py]: Makes sense. I have made the changes.

**Review by mwaddoups — APPROVED** [review]
Looks good, let's put this in

**Pull request merged**